massgov/mayflower
# Feature/dp 10215  repeated related links

## Description
Replicated related links to the bottom of the page along with changes style to the heading

## Related Issue / Ticket

- [JIRA issue DP-10215](https://jira.mass.gov/browse/DP-10215)

## Steps to Test

1. View http://localhost:3000/?p=pages-howto
2. On Desktop, ensure that:

- the links from the sidebar are replicated at the bottom of the page in the style shown in the design file:
https://eotss.invisionapp.com/share/HVNN1FRZTGQ#/screens/315579970_bottom_Sidebar_Sticky_Nav
- if there are no related links in the sidebar, no related links show in the list at the bottom of the page either

3. On Mobile, ensure that:

- the related to links from the sidebar are duplicated to the bottom of the page in the style shown in the design file:
https://eotss.invisionapp.com/share/HVNN1FRZTGQ#/screens/315579974_mobile_Sidebar
- Note: previously this was already happening, but the style has changed from the colored bar heading to the simplified heading


## Screenshots



#### @TODO
